tpm2/tpm2.go

```diff
@@ -177,6 +177,8 @@ func decodeReadPCRs(in []byte) (map[int][]byte, error) {
 }
 
 // ReadPCRs reads PCR values from the TPM.
+// This is only a wrapper over TPM2_PCR_Read() call, thus can only return
+// at most 8 PCRs digests
```

Comment:
Add a full stop here, but otherwise looks good to me!

Reply:
Done